try:
    print x
except:
    print 'error'

if 1:
    try:
        print x
    except:
        print 'error'
